woodwork/tests/logical_types/test_logical_types.py

```diff
@@ -110,6 +111,14 @@ def test_datetime_transform(datetimes):
         assert str(transform.dtype) == 'datetime64[ns]'
 
 
+def test_datetime_conversion_error():
+    series = pd.Series(['a', 'b', 'c'], name='series')
```

Comment:
Can we test with the `sample_series` fixture? Pandas and Koalas both fail as expected, but it seems Dask wouldn't show the error until a compute call is made, so probably worth xfailing there or something

Reply:
Yeah updated in 2dd47b6